src/wp-content/themes/twentyfifteen/js/functions.js

```diff
@@ -5,130 +5,162 @@
  * Contains handlers for navigation and widget area.
  */
 
-( function( $ ) {
-	var $body, $window, $sidebar, resizeTimer,
-		secondary, button;
+( function() {
+	function domReady( callback ) {
+		if ( document.readyState === 'complete' || document.readyState === 'interactive' ) {
+			callback();
+		} else {
+			document.addEventListener( 'DOMContentLoaded', callback );
+		}
+	}
+
+	function toggleClass( element, className ) {
+		if ( -1 !== element.className.indexOf( className ) ) {
+			element.className = element.className.replace( ' ' + className, '' );
+		} else {
+			element.className += ' ' + className;
+		}
+	}
+
+	var sidebar, resizeTimer, secondary, button;
 
 	function initMainNavigation( container ) {
+		if ( ! container ) {
+			return;
+		}
+
 		// Add dropdown toggle that display child menu items.
-		container.find( '.menu-item-has-children > a' ).after( '<button class="dropdown-toggle" aria-expanded="false">' + screenReaderText.expand + '</button>' );
+		Array.from( container.querySelectorAll( '.menu-item-has-children > a' ) ).forEach( function( menuLink ) {
+			var dropdownToggle = document.createElement( 'button' );
+			dropdownToggle.className = 'dropdown-toggle';
+			dropdownToggle.setAttribute( 'aria-expanded', 'false' );
+			dropdownToggle.innerHTML = screenReaderText.expand;
```

Comment:
```suggestion
			dropdownToggle.textContent = screenReaderText.expand;
```

... assuming that `screenReaderText.expand` only ever contains text and no HTML.

Reply:
It _does_ include HTML, that's why this is needed here.

Reply:
In that case, this looks good 👍